src/Tribe/Ticket_Object.php

```diff
@@ -322,7 +322,7 @@ public function date_in_range( $datetime = 'now' ) {
 			}
 
 			// Bail if we don't have an end date and the event has passed
-			if ( empty( $end ) && tribe_is_past_event() ) {
+			if ( empty( $end ) && tribe_is_past_event( tribe_events_get_event( $this->event_id ) ) ) {
```

Comment:
Shouldn't we split this into a variable? Before passing it directly into another method? Maybe check the value?

Reply:
done